pkg/metrics/server.go

```diff
@@ -4,28 +4,57 @@ import (
 	"bytes"
 	"context"
 	"crypto/sha256"
+	"crypto/tls"
+	"crypto/x509"
 	"errors"
 	"fmt"
 	"io"
+	"io/ioutil"
 	"net/http"
 	"os"
 	"time"
 
+	"k8s.io/klog/v2"
+
+	"github.com/openshift/cluster-node-tuning-operator/pkg/util"
 	"github.com/prometheus/client_golang/prometheus/promhttp"
 	"gopkg.in/fsnotify.v1"
-
-	"k8s.io/klog/v2"
 )
 
 const (
 	tlsSecretDir = "/etc/secrets"
 	tlsCert      = tlsSecretDir + "/tls.crt"
 	tlsKey       = tlsSecretDir + "/tls.key"
+
+	authCADir                = "/tmp/" + AuthConfigMapNamespace + "/" + AuthConfigMapName
```

Comment:
```suggestion
	authCADir                = "/etc/metrics-client-ca"
```
In HyperShift, we will get the CA from a secret rather than from a file created by the NTO controller. Can we change this path to something more general?